astropy/extern/plugins/pytest_doctestplus/output_checker.py

```diff
@@ -0,0 +1,199 @@
+"""
+Implements a replacement for `doctest.OutputChecker` that handles certain
+normalizations of Python expression output.  See the docstring on
+`OutputChecker` for more details.
+"""
+
+import doctest
+import re
+
+import numpy as np
+
+from ...bundled import six
+from ...bundled.six.moves import zip
+from ....utils.compat import NUMPY_LT_1_10
+
+
+_ALLCLOSE_KWARGS = {}
```

Comment:
actually we shouldn't do this and any of the numpy compatibility here. anything in externs should be verbatim of an external package release (maybe with the exception of the six imports).

Reply:
It's a little awkward in this case because I'm not sure we want to have to support `numpy<1.10` in the plugin itself, but this was necessary in order to get the backport to work. You make a very good point, though. I'm not sure what the right solution is.

Reply:
well, I'm not sure we will be able to drop that support for the LTS, so that case two version need to be maintained

Reply:
Okay, I guess we might as well add this to the plugin itself then.